www/css/style.css

```diff
@@ -907,56 +923,96 @@ button.button.back-button.buttons.button-clear.header-item {
   text-align: center; padding: 0.25em 0.8em; border-bottom-color: #ccc; border-bottom-width: 0.5px; border-bottom-style: solid;
 }
 .start-time-tag {
-        box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+        width: 68px;
+        height: 30px;
+        background: #91d4ff;
+        /* border: 1px solid #0088ce; */
+        box-sizing: border-box;
+        border-radius: 50px;
+        margin: auto;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: 'Lato';
+        font-style: normal;
+        font-weight: bold;
+        font-size: 13px;
         text-align: center;
-        background-color: #33e0bb;
         color: black;
+        position: relative;
+        bottom: -15px;
+        box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+        /* box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+        text-align: center;
+        background-color: #91d4ff;
+        color: #0088ce;
         width: 16%;
         height: 16px;
         border-radius: 8px;
         font-size: 9px;
         position: absolute;
         left: 1%;
         top: 15px;
-        line-height: 16px;
+        line-height: 16px; */
 }
 .stop-time-tag {
+        width: 68px;
+        height: 30px;
+        background: #0088ce;
+        /* border: 1px solid #0088ce; */
+        box-sizing: border-box;
+        border-radius: 50px;
+        margin: auto;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: 'Lato';
+        font-style: normal;
+        font-weight: bold;
+        font-size: 13px;
+        text-align: center;
+        color: white;
+        position: relative;
+        top: -15px;
         box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+        /* box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
         text-align: center;
-        background-color: #ff5251;
-        color: black;
+        background-color: #0088ce;
+        color: white;
         width: 16%;
         height: 16px;
         border-radius: 8px;
         font-size: 9px;
         position: absolute;
         left: 1%;
         bottom: 15px;
-        line-height: 16px;
+        line-height: 16px; */
 }
 .start-time-tag-inf-scroll {
         box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
         text-align: center;
-        background-color: #33e0bb;
+        background-color: #91d4ff;
         color: black;
         width: 16%;
-        height: 25px;
+        height: 18px;
```

Comment:
Why make all the changes to `start-time-tag` instead of `start-time-tag-inf-scroll`, when we are picking the infinite scroll for the unified view.

Reply:
This was a change that I had made prior to the unification.

Reply:
Please revert it if it is not a change that was intended to stick

Reply:
`start-time-tag-inf-scroll` is not being used, only `start-time-tag`.

`start-time-tag` looks like this:
![Simulator Screen Shot - iPhone 14 Pro Max - 2022-09-23 at 11 49 58](https://user-images.githubusercontent.com/61334340/192027414-d9d96f9a-72d8-41f2-a5b5-28bfb490357e.png)

while `start-time-tag-inf-scroll` is broken and looks like this:
![Simulator Screen Shot - iPhone 14 Pro Max - 2022-09-23 at 11 48 49](https://user-images.githubusercontent.com/61334340/192027453-d9df07ef-318e-461a-81c2-5085a28d9f4b.png)

@shankari, since `start-time-tag` works, should I just use this and delete the `start-time-tag-inf-scroll`?

Reply:
Yes, please delete `start-time-tag-inf-scroll`

Reply:
Removed this code in my commit - https://github.com/e-mission/e-mission-phone/pull/883/commits/a9d425e403fbda2c5c276992d6a23ceda673eac7